### DISCUSSION TITLE
Cross Compiler

### DISCUSSION THREAD

**Discussion by Synthbernd** [BartmanAbyss/vscode-amiga-debug#221] [Q&A] [answered]
Hello,

how can I create step by step a C Cross Compiler between Visual Studio Code for Windows and the Amiga vbcc compiler on Workbench 3.1 under Winuae?

I'm a Novice in C-Programming and installing a Cross Compiler,

**Comment by BartmanAbyss** [comment]
Hi.
I'm not sure I understand what you're trying to do.
This vscode-extension contains a cross compiler (gcc, not vbcc) and WinUAE.

**Reply by Synthbernd** [reply]
Okay how can I  install a C cross compiler between Visual Studio Code for Windows and the Amiga gcc Compiler on Workbench 3.1 in WinUAE?
Can you send me a step by step tutorial?

**Reply by BartmanAbyss** [reply, marked as answer]
Have you checked the README https://github.com/BartmanAbyss/vscode-amiga-debug/blob/master/README.md and this video https://www.youtube.com/watch?v=gQ4tKisnr7Y ?
